Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/UserAgendaRepo.kt

```diff
@@ -15,7 +15,7 @@ class UserAgendaRepo private constructor(context: Context) {
         savedSessionIds.addAll(sharedPrefs.getStringSet(sessionIdsKey, HashSet<String>()))
     }
 
-    fun isSessionBookmarked(sessionId : String) : Boolean {
+    fun isSessionBookmarked(sessionId : String?) : Boolean {
```

Comment:
There should always be a sessionId...it's non-null FYI

Reply:
done